### PR TITLE
Checks navigation <nav> tags are correctly labelled if more than one exists.

### PR DESCRIPTION
**Pull request by danspratling**
Fixes the bugs in #1 

Now, multiple navigations are allowed but should be labelled if multiple navigations exist.
Screenshot 2020-09-28 at 14 21 44

There is one downside to this approach which we can't resolve exclusively with CSS

1. We cannot add the error to the first navigation, even if the second navigation is correctly labelled (as css cannot reverse its direction).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,32 +21,28 @@ _Lightweight_ &bull; _Modern_ &bull; _Accessibile_ &bull; _Customisable_ &bull;
...
 
-A list of every a11y concern Checka11y.css will check for and highlight:
+A list of every a11y concern Checka11y.css will look for and highlight:
 
 - Missing `alt` attributes on images
 - Missing `title` on `<iframe>`
 - Checks `<li>` is the **only** child of `<ol>` and `<ul>`
 - Checks `<nav>` only exists once on any page
-- Missing `lang` on `<html>`
-- Checks `<a target="_blank">` and warns about links that open in a new window
```
Comment (jackdomleo7) [inline README.md:41]: Could we add these back 😉

File: README.md [hunk 2]
```diff
@@ -21,32 +21,28 @@ _Lightweight_ &bull; _Modern_ &bull; _Accessibile_ &bull; _Customisable_ &bull;
...
 
 Our features are always checked against the W3C accessibility guidelines.
 
-A list of every a11y concern Checka11y.css will check for and highlight:
+A list of every a11y concern Checka11y.css will look for and highlight:
 
 - Missing `alt` attributes on images
 - Missing `title` on `<iframe>`
 - Checks `<li>` is the **only** child of `<ol>` and `<ul>`
 - Checks `<nav>` only exists once on any page
```
Comment (jackdomleo7) [inline README.md:39]: Can this be modified to match more closely to your work? 😊

File: README.md [hunk 3]
```diff
@@ -21,32 +21,28 @@ _Lightweight_ &bull; _Modern_ &bull; _Accessibile_ &bull; _Customisable_ &bull;
...
+[![GitHub stars](https://img.shields.io/github/stars/jackdomleo7/Checka11y.css?style=social)](https://github.com/jackdomleo7/Checka11y.css/stargazers "Total GitHub stars") [![GitHub release (latest by date)](https://img.shields.io/github/v/release/jackdomleo7/Checka11y.css?style=social)](https://github.com/jackdomleo7/Checka11y.css/releases "Current version/release") [![npm](https://img.shields.io/npm/dt/checka11y-css?logo=npm&style=social)](https://www.npmjs.com/package/checka11y-css "Total npm downloads") ![jsDelivr hits (npm)](https://img.shields.io/jsdelivr/npm/hm/checka11y-css?label=JsDelivr&logo=jsdelivr&style=social "Monthly JsDelivr CDN hits")
 
 ---
 
 ## Features
 
 Our features are always checked against the W3C accessibility guidelines.
 
-A list of every a11y concern Checka11y.css will check for and highlight:
+A list of every a11y concern Checka11y.css will look for and highlight:
```
Comment (jackdomleo7) [inline README.md:34]: `look` or `check`?

File: README.md [hunk 4]
```diff
@@ -21,32 +21,28 @@ _Lightweight_ &bull; _Modern_ &bull; _Accessibile_ &bull; _Customisable_ &bull;
...
 - Checks `<nav>` only exists once on any page
-- Missing `lang` on `<html>`
-- Checks `<a target="_blank">` and warns about links that open in a new window
-- Checks `<dt>` and `<dd>` are the **only** children of `<dl>`
 
 Other features:
 
 - Clear a11y errors and warnings with informative messages
 - Options to import via CDN, npm or yarn
-- Customise the error and warning colors
```
Comment (jackdomleo7) [inline README.md:48]: Could these be added back in?

File: test/index.html [hunk 5]
```diff
@@ -42,6 +42,7 @@ <h2>Lists: All children of &lt;ul&gt; must only be &lt;li&gt;</h2>
     <section>
       <h2>Navigation: Only a single navigation element should exist</h2>
       <nav>This is the first navigation</nav>
+      <p>This is separating content</p>
```
Comment (jackdomleo7) [inline test/index.html:45]: Could you screenshot this and add to the PR description?

File: checka11y.css [hunk 6]
```diff
@@ -82,14 +85,15 @@ dl > *:not( dt ):not( dd )::after {
  * Navigation
  */
 
-/* There must only be one primary navigation on the page */
-nav + nav {
+/* There if more than one primary navigation is available, all navigations must be clearly labelled. */
```
Comment (jackdomleo7) [inline checka11y.css:88]: Is the first `There` needed?

File: checka11y.css [hunk 7]
```diff
@@ -11,6 +11,8 @@
 }
 
 /**
+<<<<<<< HEAD
```
Comment (jackdomleo7) [inline checka11y.css:14]: Could we remove these lines?

File: checka11y.css [hunk 8]
```diff
@@ -26,6 +28,7 @@ html:not( [lang] ) body::before {
 }
 
 /**
+>>>>>>> master
```
Comment (jackdomleo7) [inline checka11y.css:31]: Could we remove this line?

**Review by jackdomleo7 — CHANGES_REQUESTED** [review]
Have to admit, this a new thing to me so I'm learning about multiple navs as you write this 😉 but looks good to me, just a few comments 😊

File: checka11y.css (hunk 8 above)
Comment (danspratling) [reply checka11y.css:31]: Wait how did these diffs sneak in there? 😅

File: README.md (hunk 3 above)
Comment (danspratling) [reply README.md]: It doesn't matter. This was a change you made and I just accidentally overwrote the readme.

**Comment by danspratling** [comment]
Resolved the issues which were there. I think I merged the wrong readme when there was a conflict.

Also added a screenshot above.

**Review by jackdomleo7 — APPROVED** [review]
Nice! Feel free to squash and merge 😊

**Pull request merged**